examples/abstract-objects/abstract_objects.bal

```diff
@@ -30,21 +30,17 @@ type Employee object {
         self.lastName = lastName;
     }
 
-    // Methods should have a body either within the object or as outside method definitions.
+    // Methods should have a body.
```

Comment:
```suggestion
    // Methods should have a body.
```
Why did we remove that part?

Reply:
Method body outside object is no longer supported.